dataprofiler/data_readers/csv_data.py

```diff
@@ -100,13 +100,31 @@ def _check_and_return_options(self, options):
         :return: None
         """
         options = super()._check_and_return_options(options)
-        for key, value in options.items():
-            if key == 'header' and value != 'auto' and value is not None \
-                    and not (isinstance(value, int) and value > -1):
+        
+        if 'header' in options.keys():
+            value = options["header"]
+            if value != 'auto' and value is not None \
+                and not (isinstance(value, int) and value > -1):
```

Comment:
@gme5078 Do you think I should remove the `value = ...` line and just collapse it into a single if statement? I wasn't sure which one was more readable

Reply:
In my opinion, thats a bit much, and I think this is more readable